frontend/app/states/play.js

```diff
@@ -79,10 +80,32 @@ class PlayState extends Phaser.State {
       this.game.world.width / 2.5, 0,
       this.game.camera.width / 100, this.game.world.height - this.game.ground.height);
 
-    this.test = new Modal(this.game, new Keyboard(this.game));
+    //add keyboard demo button
+    this.keyboardDemoButton = new Button(this.game, null, 0,
+        this.game.world.height - 135, this.game.camera.width/12,
```

Comment:
Also technically `0` is a hard-coded pixel value here too but we probably shouldn't be shoving the button all the way against the side of the world anyway.